.Documentation/articles/step-by-step-guides/hello-creator.md

```diff
@@ -0,0 +1,120 @@
+# Hello Creator – a 5-step guide to a basic training application
+
+You have successfully downloaded and imported the Innoactive Creator. This guide will help you to create your first basic training application in 5 steps. During the steps, we introduce you to the basic user interface of the Unity Editor and the Innoactive Creator and explain basic concepts of a successful creation of a training application using the Creator.
+
+## Step 1: Create Course
+
+In case you are new to Unity, you see an empty Unity project, which usually includes a **scene** window, a **hierarchy** of 3D objects of your scene which is currently empty and only contains a `Main Camera` and `Directional Light`, the Inspector (might also be empty currently), and on the lower end of the Unity window, a **project hierarchy and console** window (see figure 1).
+
+![Unity Layout](../images/step-by-step-guides/unityWindows.jpg "Getting Familiar with Unity - The Unity Layout")
+*Figure 1: Get familiar with Unity windows: hierachy, scene, inspector and project hierachy and console.*
+
+By importing Innoactive Creator, you will have a new element in the top menu (‘File’, ’Edit’, …), called `Innoactive` (see Fig. 2).
+
+> select `Create New Course` from the top menu `Innoactive`.
+
+## Step 2: Select Demo Scene
+
+A **Wizard** window opens.
```

Comment:
This should happen automatically when the Creator is imported into a new project

Reply:
cc @JulieAtInnoactive